docs/migration-guide.rst

```diff
@@ -159,6 +159,19 @@ place (in the Girder python package) in both development and production installs
 The built assets are installed into a virtual environment specific static path
 ``{sys.prefix}/share/girder``.
 
+Static root is required during web client build
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The static root, indicating the base URL where web client files are served from,
+is now required when the web client is built. The primary implication is that the static root
```

Comment:
`The primary implication is that the static root` 
->
`The primary implication is that if the static root`

Reply:
Will fix.